bugbug/models/bug.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, lemmatization=False):
             bug_features.landings(),
             bug_features.title(),
             bug_features.comments(),
+            bug_features.cleanup_url(),
```

Comment:
Sorry for the confusion, I didn't mean to make it another feature extractor. I just meant we should have "cleanup" functions implemented similarly to feature extractors.

Reply:
So, call the cleanup functions in the place where you were calling it before, but making it more generic.

Reply:
Something like:
```
for cleanup_function in cleanup_functions:
    summary = cleanup_function(summary)
    comments = cleanup_function(comments)
```